presto-native-execution/presto_cpp/main/PrestoServerOperations.h

```diff
@@ -27,21 +28,30 @@ struct ServerOperation;
 /// Static class implements Presto Server Operations.
 class PrestoServerOperations {
  public:
-  static void runOperation(
+  PrestoServerOperations(TaskManager* const taskManager)
+      : taskManager_(taskManager) {}
+
+  void runOperation(
       proxygen::HTTPMessage* message,
       proxygen::ResponseHandler* downstream);
 
-  static std::string connectorOperation(
+  std::string connectorOperation(
       const ServerOperation& op,
       proxygen::HTTPMessage* message);
 
-  static std::string systemConfigOperation(
+  std::string systemConfigOperation(
       const ServerOperation& op,
       proxygen::HTTPMessage* message);
 
-  static std::string veloxQueryConfigOperation(
+  std::string veloxQueryConfigOperation(
       const ServerOperation& op,
       proxygen::HTTPMessage* message);
+
+  std::string debugOperation(
+      const ServerOperation& op,
+      proxygen::HTTPMessage* message);
+
+  TaskManager* const taskManager_;
```

Comment:
Should this be a shared_ptr?

Reply:
Current codebase stores TaskManager as a unique ptr initially, and then raw pointer in several other classes. So just following that